### PR TITLE
Added qr code generation

### PR DESCRIPTION
**Pull request by joshterrill**
Related to #94, added qr code generation upon torrent file selection. Maybe we want to make the QR code larger? Or position it somewhere else? I figured this was fine for right now.

### CODE REVIEW

File: client/index.js [hunk 1]
```diff
@@ -166,6 +167,10 @@ function onTorrent (torrent) {
     '<a href="' + torrent.magnetURI + '" target="_blank">[Magnet URI]</a> ' +
     '<a href="' + torrent.torrentFileBlobURL + '" target="_blank" download="' + torrentFileName + '">[Download .torrent]</a>'
   )
+  
+  util.log(
+    'QR Code: <img class="qr" src="data:image/png;base64,' + qr.imageSync(location.href + '#' + torrent.infoHash, { type: 'png' }).toString('base64') + '">'
```
Comment (feross) [inline client/index.js]: Small bug: If there's already a hash in the URL, this will add a second one.

**Comment by feross** [comment]
You don't need to commit the `qrcode.min.js` file, since you're already using `require` to include the package.

**Comment by feross** [comment]
One thing that's a bit of a bummer is that this single dependency almost doubles the size of the JS bundle.

```
cat static/bundle.js | gzip | wc -c
  144612
browserify -r node-qr-image | gzip | wc -c
  103611
```

**Comment by feross** [comment]
Other than my comments, this looks good!

If you know another dependency that's a bit lighter weight than this, let's swap it in.

**Comment by joshterrill** [comment]
Actually that js file shouldn't even be there. My bad. That was for another
one I was testing out. And yeah ill look for another dependency... most of
the other ones I looked at required some sort of C libraries that I didn't
have installed so I figured it would be bad to use those. But I will still
look around.

**Comment by feross** [comment]
No problem if you can't find one. We can still merge this. But there's no way that QR generation can require that much code.

**Comment by feross** [comment]
It looks like most of the size of `node-qr-image` comes from bundling a zlib library. We should just find a library that uses canvas to draw the image. You can even get a PNG from a canvas without manually making the PNG yourself ;)

**Comment by joshterrill** [comment]
Woops, forgot this was here. I'll take care of this and fix what needs to be fixed this week.

**Comment by feross** [comment]
Hey @joshterrill, no pressure, just wondering if you plan to finish up this PR? If not, no worries. Just let me know.

**Comment by joshterrill** [comment]
Sure I can do it. What do you think about using something like this? https://github.com/alexeyten/qr-image

**Comment by feross** [comment]
Looks like a good library to me.

**Pull request closed without merging**